relation_engine/batchload/test/time_travelling_database_test.py

```diff
@@ -735,6 +737,76 @@ def test_batch_update_vertex_fail_not_vertex_collection(arango_db):
     _check_exception(lambda: b.create_vertex('i', 'v', 6, {}), ValueError,
         'Batch updater is configured for an edge collection')
 
+def test_batch_create_edges(arango_db):
+    """
+    Test creating 2 vertices in one batch.
+    """
+    col = arango_db.create_collection('v')
+    arango_db.create_collection('e', edge=True)
+    att = ArangoBatchTimeTravellingDB(arango_db, 'v', default_edge_collection='e')
+    
+    b = att.get_batch_updater('e')
+
+    key = b.create_edge(
+        'id1',
+        {'id': 'whee', '_id': 'v/1'},
```

Comment:
Are these edges in the vertex collection? I see that create_edges says they must be, but i'm not sure where these are defined as in the vertex collection in this test.

Reply:
They're basically a mock. The vertexes need to be pulled from the DB to ensure that the `_id` key exists, and I want to hide that key from the delta load algorithm as it's an implementation detail. So normally those would be full vertex documents but I'm cheating here. I'll add some comments to clarify.